gatsby-browser.js

```diff
@@ -8,3 +8,7 @@ import "./src/style.css"
 
 // Highlighting for code blocks
 import "prismjs/themes/prism.css"
+
+export const shouldUpdateScroll = () => {
```

Comment:
This is an important magical line of code that forces the page scroll to "jump" when we switch between painting categories. This creates an illusion of opening a new page.